index.js

```diff
@@ -81,7 +81,7 @@ function testGenResponse(swagger, path, operation, response, config) {
     operation + '.handlebars');
 
   source = read(templatePath, 'utf8');
-  templateFn = handlebars.compile(source);
+  templateFn = handlebars.compile(source, {noEscape: true});
```

Comment:
if i don't use this flag, the html codes replace the actual character. before i added this, all of the apostrophes were some thing like '''&amp;#26d''' or something
then when i added that flag, it made them apostrophes again